src/README.md

```diff
@@ -0,0 +1,75 @@
+# Use of LabVIEW Component Simulator
+
+The Component Simulator allows you to call a LabVIEW TCP Server to test the TCP communication between the TCP Server and the TCP Client. 
+The LabVIEW TCP Server receives json commands from a Python TCP Client, and send Ack and Events back to the Client, using TCP packets.
+
+## Needed Package
+
+- Caraya 1.0.2.115 (for Unit Testing)
+
+## Use of TCP Server
+
+### Build the Executable
+
+The LabVIEW professional version is needed to build the executable. 
+You can build the executable from within LabVIEW or from the command line, as you prefer:
+
+- From within LabVIEW: select the build all under Build Specifications of tcpip.lvproj. The ComponentSimulator executable will be in the `build/` directory.
+The output log is in the build/buildComponentSimulator.txt.
+- From the command line:
+```sh
```

Comment:
I think there should be a new line before the code block. You could check the syntax by the vscode (with the plugin). There is the similar problems in other places as well.

Reply:
Done.